content/en/docs/tasks/security/automtls/index.md

```diff
@@ -0,0 +1,276 @@
+
+{{< text bash >}}
+$ kubectl exec $(kubectl get pod -l app=sleep -n mixed -o jsonpath={.items..metadata.name}) -c sleep -n mixed -- curl http://httpbin.foo:8000/ip -s -o /dev/null -w "%{http_code}\n"
+200
+{{< /text >}}
+
+This one-liner command conveniently iterates through all reachability combinations:
+
+{{< text bash >}}
+$ for from in "foo" "mixed" "legacy"; do for to in "foo" "mixed" "legacy"; do kubectl exec $(kubectl get pod -l app=sleep -n ${from} -o jsonpath={.items..metadata.name}) -c sleep -n ${from} -- curl "http://httpbin.${to}:8000/ip" -s -o /dev/null -w "sleep.${from} to httpbin.${to}: %{http_code}\n"; done; done
+sleep.foo to httpbin.foo: 200
+sleep.foo to httpbin.mixed: 200
+sleep.foo to httpbin.legacy: 200
+sleep.mixed to httpbin.foo: 200
+sleep.mixed to httpbin.mixed: 200
+sleep.mixed to httpbin.legacy: 200
+sleep.legacy to httpbin.foo: 200
+sleep.legacy to httpbin.mixed: 200
+sleep.legacy to httpbin.legacy: 200
+{{< /text >}}
+
+In the setup, we start with `PERMISSIVE` for all services in the mesh. Combined with automatic
+mutual TLS enabled, sleep from `foo` and `mixed` client, send mutual TLS traffic to `httpbin.foo`
+service and plain text to `httpbin.legacy` service. For `httpbin.mixed`, the client sidecar send
+mutual TLS to the pod with sidecar injected deployment, and plain text to the pod without sidecar.
+
+### Configure mutual TLS STRICT
+
+The default installation configures all the service in `PERMISSIVE` mode by default, which accepts
+both plain text and mutual TLS traffic. Now we configure `httpbin.foo` service to mutual TLS
+`STRICT` mode, only accepting mutual TLS traffic.
+
+{{< text bash >}}
+$ cat <<EOF | kubectl apply -n foo -f -
+apiVersion: "authentication.istio.io/v1alpha1"
+kind: "Policy"
+metadata:
+  name: "httpbin"
+spec:
+  targets:
+  - name: httpbin
+  peers:
+  - mtls: {}
+EOF
+{{< /text >}}
+
+Now the requests from the clients without sidecar start to fail.
+
+{{< text bash >}}
+$ for from in "foo" "mixed" "legacy"; do for to in "foo" "mixed" "legacy"; do kubectl exec $(kubectl get pod -l app=sleep -n ${from} -o jsonpath={.items..metadata.name}) -c sleep -n ${from} -- curl "http://httpbin.${to}:8000/ip" -s -o /dev/null -w "sleep.${from} to httpbin.${to}: %{http_code}\n"; done; done
+sleep.foo to httpbin.foo: 200
+sleep.foo to httpbin.mixed: 200
+sleep.foo to httpbin.legacy: 200
+sleep.mixed to httpbin.foo: 200
+sleep.mixed to httpbin.mixed: 200
+sleep.mixed to httpbin.legacy: 200
+sleep.legacy to httpbin.foo: 000
+command terminated with exit code 56
+sleep.legacy to httpbin.mixed: 200
+sleep.legacy to httpbin.legacy: 200
+{{< /text >}}
+
+### Disable mutual TLS to plain text
+
+Now we can explicitly disable mutual TLS to plain text for `httpbin.foo`.
+
+{{< text bash >}}
+$ cat <<EOF | kubectl apply -n foo -f -
+apiVersion: "authentication.istio.io/v1alpha1"
+kind: "Policy"
+metadata:
+  name: "httpbin"
+spec:
+  targets:
+  - name: httpbin
+EOF
```

Comment:
Would be good to add the fields that explicitly disable, rather than relying on default.

Reply:
We no longer have the plaintext option in authentication policy. This is the only way users can disable mTLS today.